assets/js/app.js

```diff
@@ -345,8 +357,18 @@ global.MonsieurBizShippingSlotManager = class {
   }
 
   getShippingSlotConfigSelect(shippingMethodCode) {
-    return Array.from(this.shippingSlotConfigSelects).find(
-      shippingSlotConfigSelect => shippingSlotConfigSelect.name.includes(shippingMethodCode)
-    ) ?? null;
+    for (let shippingSlotConfigContainer of this.shippingSlotConfigContainers) {
+      let shippingSlotConfigSelect = shippingSlotConfigContainer.querySelector('select[name*="[shippingSlotConfigs]"]');
+      console.log(shippingSlotConfigContainer, shippingSlotConfigSelect, shippingMethodCode, shippingSlotConfigSelect.name.includes(shippingMethodCode), shippingSlotConfigSelect.name);
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏Remove the log